modules/unpickler/src/test/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3UnpicklerStats.scala

```diff
@@ -136,7 +136,9 @@ class Scala3UnpicklerStats extends munit.FunSuite:
         sym match
           case Some(t) =>
             counter.addSuccess(mthd)
-            sym
+            sym match
+              case None => None
+              case Some(t) => t.symbol
```

Comment:
Not sure why we need to return anything but:
```suggestion
            Some(t.symbol)
```